pkg/action/rollback.go

```diff
@@ -42,6 +42,8 @@ type Rollback struct {
 	Recreate      bool // will (if true) recreate pods after a rollback.
 	Force         bool // will (if true) force resource upgrade through uninstall/recreate if needed
 	CleanupOnFail bool
+	// MaxHistory limits the maximum number of revisions saved per release
+	MaxHistory int
```

Comment:
I'm on my phone so I could be wrong, but this doesn't look like it's properly formatted.  Could you run `make test-style` to confirm?

Reply:
Thx,I put the comment at the end of the line so it looks normal